package.json

```diff
@@ -94,26 +86,27 @@
     "electron-icon-maker": "^0.0.4",
     "esm": "^3.2.25",
     "fancy-log": "^1.3.3",
+    "globby": "^13.1.1",
     "gulp": "^4.0.0",
     "html-webpack-plugin": "^5.1.0",
     "husky": "^1.3.1",
+    "intl-messageformat": "^9.12.0",
```

Comment:
We use this in our web code.

Reply:
Ah, duh

Reply:
Right now the only import I see is in the storybook:

https://github.com/Jigsaw-Code/outline-client/blob/8016293e1a0f7b58754f076359065a9670c37434/src/www/.storybook/localize.ts#L18-L19

Reply:
You're right. We use it in the Manager, I think